azurerm/internal/services/compute/windows_virtual_machine_resource.go

```diff
@@ -1175,19 +1175,20 @@ func resourceWindowsVirtualMachineDelete(d *schema.ResourceData, meta interface{
 		return fmt.Errorf("retrieving Windows Virtual Machine %q (Resource Group %q): %+v", id.Name, id.ResourceGroup, err)
 	}
 
-	// ISSUE: XXX
-	// shutting down the Virtual Machine prior to removing it means users are no longer charged for the compute
-	// thus this can be a large cost-saving when deleting larger instances
```

Comment:
(as above) can we add this back in - details are in https://github.com/terraform-providers/terraform-provider-azurerm/issues/4920

Reply:
will add the same comment on both vm files